editor/src/app/sites/sections/sections-state/site-sections.state.ts

```diff
@@ -263,14 +263,15 @@ export class SiteSectionsState implements NgxsOnInit {
   reOrderSiteSections({ getState, setState }: StateContext<SiteSectionStateModel[]>, action: ReOrderSiteSectionsAction) {
     const siteName = this.store.selectSnapshot(AppState.getSite);
     const sectionsToSort = this.store.selectSnapshot(SiteSectionsState.getCurrentSiteSections);
+    const isFirefox = window.navigator.userAgent.indexOf('Firefox');
 
     sectionsToSort.sort((sectionA, sectionB) => {
       if (sectionA.order !== action.currentOrder && sectionB.order !== action.currentOrder) {
-        return sectionB.order > sectionA.order ? -1 : 1;
+        return isFirefox !== -1 ? (sectionB.order >= sectionA.order ? -1 : 1) : (sectionB.order > sectionA.order ? -1 : 1);
```

Comment:
It's not OK to do comparisons based on the browser. There should be a code that works the same across browsers. Look for a different solution.

Also, this piece of code needs some cleanup. When doing multiple comparisons of variables one below another, it's better to always keep them on the same sides, otherwise, it's hard to understand.

this:
```
sectionB.order > action.payload
sectionA.order < action.payload
```

instead of this:
```
sectionB.order > action.payload
action.payload > sectionA.order 
```